indico/modules/events/abstracts/blueprint.py

```diff
@@ -42,6 +44,8 @@
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
                  RHAbstractsDownloadAttachment)
+_bp.add_url_rule('/abstracts/mine', 'my_abstracts', RHMyAbstracts)
+_bp.add_url_rule('/abstracts/my-abstracts.pdf', 'my_abstracts_pdf_export', RHMyAbstractsExportPDF)
```

Comment:
How about `my_abstracts_pdf` instead of `my_abstracts_pdf_export`? The `_export` seems pretty redundant, the name is clear without it too.